content/sensu-core/0.29/guides/securing-rabbitmq-guide.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Securing RabbitMQ"
+description: "Strategies and best practices for securing RabbitMQ"
+version: "0.29"
+weight: 9
+previous: ../securing-sensu-guide
+next: ../securing-redis-guide
+menu:
+  sensu-core-0.29:
+    parent: guides
+---
+
+As the supported transport mechanism for any Sensu deployment, RabbitMQ has its own set of security concerns, from ensuring that VHOST permissions are set correctly, to adding SSL/TLS encryption between clients, consumers, and queues. This guide will discuss how to properly secure RabbitMQ as one of the core elements of a Sensu deployment.
+
+Before we dive too deep in the article, you may want to familiarize yourself with [RabbitMQ permissions][1]. It's important to note that RabbitMQ makes a distinction between _configure_, _write_, and _read_ permissions. With that in mind, let's move on to what we'll cover in this guide.
+
+# Objectives
+
+This guide will discuss the following:
+
+* [VHost Permissions for Sensu Clients](#vhost-permissions-clients)
+* [VHost Permissions for Sensu Servers](#vhost-permissions-servers)
+* [SSL/TLS Configuration](#vhost-permissions-clients)

```

Comment:
Should be: `* [SSL/TLS Configuration](#ssl-tls-configuration)`

Reply:
:heavy_check_mark: